fe/fe-core/src/main/java/org/apache/doris/nereids/StatementContext.java

```diff
@@ -40,12 +43,12 @@ public class StatementContext {
     private StatementBase parsedStatement;
 
     public StatementContext() {
-        this.connectContext = ConnectContext.get();
+        setConnectContext(ConnectContext.get());
     }
 
     public StatementContext(ConnectContext connectContext, OriginStatement originStatement) {
-        this.connectContext = connectContext;
-        this.originStatement = originStatement;
+        setConnectContext(connectContext);
+        setOriginStatement(originStatement);
```

Comment:
why change to set?

Reply:
It's just for adding channel to statementContext, I will remove it if useless.